contributed_definitions/NXenergydispersion.nxdl.xml

```diff
@@ -69,19 +69,79 @@
+             Way of scanning the energy axis
         </doc>
         <enumeration>
-            <item value="fixed"/>
-            <item value="sweep"/>
+            <item value="fixed_analyser_transmission">
+                <doc>
+                     constant ΔE mode, where the electron retardation (i.e., the fraction of pass energy to
+                     kinetic energy, R = (EK - WF/Ep)) is scanned, but the pass energy Ep is kept constant.
+                     Here, WF is the spectrometer work function.
+                     This mode is often used in XPS/UPS because the energy resolution does not change with
+                     changing energy (due to the constant pass energy).
+                     
+                     Synonyms: constant ΔE mode, constant analyser energy mode, CAE mode, FAT mode
+                     
+                         This concept is related to term `12.64`_ of the ISO 18115-1:2023 standard.
+                     .. _12.64: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.64
+                </doc>
+            </item>
+            <item value="fixed_retardation_ratio">
+                <doc>
+                     constant ΔE/E mode, where the pass energy is scanned such that the electron retardation
+                     ratio is constant. In this mode, electrons of all energies are decelerated with this same
+                     fixed factor. Thus, the pass energy is proportional to the kinetic energy. This mode is often
+                     used in Auger electron spectroscopy (AES) to improve S/N for high-KE electrons, but this
+                     leads to a changing energy resolution (∆E ~ Ep) at different kinetic energies. It can however
+                     also be used in XPS.
+                     
+                     Synonyms: constant ΔE/E mode, constant retardation ratio mode, CRR mode, FRR mode
+                     
+                         This concept is related to term `12.66`_ of the ISO 18115-1:2023 standard.
+                     .. _12.66: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.66
+                </doc>
+            </item>
+            <item value="fixed_energy">
+                <doc>
+                     In the fixed energy (FE) mode, the intensity for one single kinetic energy is measured for a
+                     specified time. This mode is particulary useful during setup or alignment of the
+                     electron analyzer, for analysis of stability of the excitation source or for sample
+                     alignment.
+                     
+                     Since the mode measures intensity as a function of time, the difference in channel signals 
+                     is not of interest. Threfore, the signals from all channels are summed.
+                     
+                     Synonyms: FE mode
+                </doc>
```

Comment:
If I understand this correctly, this does not produce an energy spectrum, and therefore is not so much of relevance for NXmpes. maybe we should restrict the enumeration there?

Reply:
For me, this would be pretty helpful to keep because this is the way I perform my alignment scans. These are typically saved in the same file as the actual data. Alignment scans are measurement points vs. intensity at one fixed kinetic energy.

Reply:
sure, fine, but if we require energy axis these won't be compatible with NXmpes

Reply:
Can we not just have a 0D energy axis, i.e. simply write the fixed energy that was used for alignment?

I am hesitant to remove this since at some point, we will not have the possibility to write some portion of data that typically occurs in MPES-related experiments (with the machines our vendors produce) with our definition. Same as for the PEEM app def.